ui/apps/platform/src/Containers/AccessControl/AuthProviders/authProviders.utils.ts

```diff
@@ -212,5 +212,5 @@ export function getDefaultRoleByAuthProviderId(groups: Group[], id: string): str
     if (defaultRoleGroups.length) {
```

Comment:
Global default is not a thing any more. If there is no default role, we should use `Undefined, assuming None`.